webpack.build.js

```diff
@@ -22,12 +22,12 @@ function rxjsExternal(context, request, cb) {
 module.exports = [{
 	devtool: "source-map",
 	entry: {
-		neutrino: "./src/index.ts",
+		"carbon-angular": "./src/index.ts",
```

Comment:
This should actually be 

```suggestion
		carbonAngular: "./src/index.ts",
```

Because I think it's used to generate some names people can use ... same with the library definition below